app/src/main/java/com/nononsenseapps/feeder/archmodel/SettingsStore.kt

```diff
@@ -138,6 +138,7 @@ class SettingsStore(
                 saved = sp.getBoolean(PREFS_FILTER_SAVED, false),
                 recentlyRead = sp.getBoolean(PREFS_FILTER_RECENTLY_READ, true),
                 read = sp.getBoolean(PREFS_FILTER_READ, true),
+                search = "",
```

Comment:
This is not the right place for this. The search query is not a setting.

Should probably be a mutableState in Repository.kt, considering that it is needed later when passed to FeedItemStore